src/stylesheets/main.css

```diff
@@ -43,11 +54,11 @@ main.container {
 
 .navbar {
   --bs-navbar-padding-y: 0;
-  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.04);
 }
 
 .navbar-nav {
-  --bs-nav-link-color: #212121;
+  --bs-nav-link-color: var(--bs-body-color);
+  box-shadow: 0px 4px 4px var(--calitp-cyan-1);

```

Comment:
Oh hmm, I think this should be in `.navbar`'s styles?

I'm seeing this when the style is in `.navbar-nav`

![image](https://github.com/cal-itp/calitp.org/assets/25497886/b6fcc5f6-eeaf-4985-b708-e9c6d0bd88b9)

Reply:
Oops! Hah. Okay here is the _real_ fix 😅 

<img width="1512" alt="image" src="https://github.com/cal-itp/calitp.org/assets/3673236/8a26abcd-0cb6-480b-9b2a-e10de9f8f633">